decidim-meetings/lib/decidim/meetings/engine.rb

```diff
@@ -50,6 +50,14 @@ class Engine < ::Rails::Engine
         end
       end
 
+      initializer "decidim_meetings.content_security_handlers" do |_app|
```

Comment:
We should probably separate between the ones that are from meetings (like youtube and jtisi) and others that are generic (i.e. to decidim-core or even the default policy, like youtube and vimeo), as they're video embeds supported by the WYSIWYG editor and could potentially be used on an installation without `decidim-meetings` enabled (see #10986)

Reply:
@andreslucena, thanks for the recommendation. This is what i had in mind, when i have commented [here](https://github.com/decidim/decidim/pull/11155#discussion_r1272993585)